### PR TITLE
23 make existing documents editable

### PR DESCRIPTION
**Pull request by EEtele**
*what I've already said*
+ merged with master

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -51,7 +51,7 @@ android {
 
 dependencies {
     coreLibraryDesugaring 'com.android.tools:desugar_jdk_libs:1.2.2'
-
+    implementation("com.himanshoe:kalendar:1.1.0")
```
Comment (EmmChriss) [inline app/build.gradle:54]: This library is already added as a dependency

File: app/src/main/java/com/lab/skuld/ui/App.kt [hunk 2]
```diff
@@ -39,16 +37,56 @@ import androidx.compose.ui.Modifier
...
+
+    return Event(
+        maybe.id,
+        maybe.startDate?.toDate(),
+        maybe.endDate?.toDate(),
+        maybe.title,
+        maybe.checked,
+        maybe.contents
+    )
+}
```
Comment (EmmChriss) [inline app/src/main/java/com/lab/skuld/ui/App.kt:87]: Please don't clutter the main compose file, if you need a class from another file, import it, or move it into a standalone file in `model` package

File: app/src/main/java/com/lab/skuld/ui/screens/NewNoteScreen.kt [hunk 3]
```diff
@@ -22,45 +24,66 @@ import androidx.compose.runtime.getValue
...
-
-
-
-
-
+data class TextData(var index: Int, var header: String, var value: String)
+data class Document(var header: String= "Title", var image: Painter? = null, var  documentContents: List<TextData> = listOf())
 @Composable
-fun ShowNewNoteScreen(){
+fun ShowNewNoteScreen(documentt: Event = event){
```
Comment (EmmChriss) [inline app/src/main/java/com/lab/skuld/ui/screens/NewNoteScreen.kt:61]: typo

File: app/src/main/java/com/lab/skuld/ui/screens/NewNoteScreen.kt [hunk 4]
```diff
@@ -22,45 +24,66 @@ import androidx.compose.runtime.getValue
...
+    var documentt = Event(id = "123", startDate = Date(), endDate = null, title = "My Event", checked = null, contents = "Line 1\nLine 2\nLine 3")
+    var document = eventToDocument(documentt)
```
Comment (EmmChriss) [inline app/src/main/java/com/lab/skuld/ui/screens/NewNoteScreen.kt:63]: or not?

**Review by EmmChriss — CHANGES_REQUESTED** [review]

**Review by EEtele — COMMENTED** [review]
Fair enough

File: app/src/main/java/com/lab/skuld/ui/screens/NewNoteScreen.kt (hunk 4 above)
Comment (EEtele) [reply app/src/main/java/com/lab/skuld/ui/screens/NewNoteScreen.kt:63]: probably not

**Pull request merged**